scripts/install/InitMetadataService.php

```diff
@@ -60,8 +62,8 @@ public function __invoke($params)
             }
         } else {
             $importerConfig = [
-                MetadataImporter::INJECTOR_KEY     => [],
-                MetadataImporter::EXTRACTOR_KEY    => [],
+                MetadataImporter::INJECTOR_KEY     => [\oat\taoQtiItem\model\qti\metadata\ontology\LomInjector::class],
```

Comment:
For readability it might be a good idea to import `\oat\taoQtiItem\model\qti\metadata\ontology\LomInjector`